tool.specs.json

```diff
@@ -6348,6 +6348,19 @@
         "type": "object"
       }
     },
+    {
```

Comment:
Did you add it manually? This `spec` is auto-generated every commit to main, you don't need to generate that manually